src/debugger/channel/DebugMessage.js

```diff
@@ -24,14 +24,18 @@ export class DebugMessage {
   static BREAKPOINT_ENABLE_COMMAND: string = "Breakpoint-enable-command";
   // Command to disable a breakpoint
   static BREAKPOINT_DISABLE_COMMAND: string = "Breakpoint-disable-command";
+  // Command to fetch stack frames
+  static STACKFRAMES_COMMAND: string = "Stackframes-command";
 
   /* Messages from Prepack to adapter */
   // Respond to the adapter that Prepack is ready
   static PREPACK_READY_RESPONSE: string = "PrepackReady";
   // Respond to the adapter that Prepack is finished
   static PREPACK_FINISH_RESPONSE: string = "PrepackFinish";
   // Respond to the adapter that Prepack has stopped on a breakpoint
-  static BREAKPOINT_STOPPED_RESPONSE: string = "Breakpoint-stopped";
+  static BREAKPOINT_STOPPED_RESPONSE: string = "Breakpoint-stopped-response";
+  // Respond to the adapter with the request stackframes
+  static STACKFRAMES_RESPONSE: string = "Stackframes-response";
```

Comment:
What is the difference between response and acknowledge?

Reply:
Acknowledge is just notifying the adapter that Prepack has gotten the request. Response is when Prepack sends information requested by the adapter. Currently, there aren't any requests that use both, but there will be in the future.

Reply:
Add a space to separate these three categories and add comments to explain the meaning of each category.